content/influxdb/cloud-iox/migrate-data/_index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Migrate data to the InfluxDB IOx storage engine
+description: >
+  Migrate data from InfluxDB backed by TSM (OSS, Enterprise, or Cloud) to
+  InfluxDB Cloud backed by InfluxDB IOx.
+menu:
+  influxdb_cloud_iox:
+    name: Migrate data
+weight: 9
+---
+
+Migrate data to InfluxDB Cloud backed by InfluxDB IOx from other 
+InfluxDB instances backed by TSM including InfluxDB OSS 1.x, 2.x,
+InfluxDB Enterprise, and InfluxDB Cloud.
+
+- [Should you migrate?](#should-you-migrate)
+  - [Do you not want to worry about series cardinality?](#do-you-not-want-to-worry-about-series-cardinality)
+  - [Do you want to use SQL to query your data?](#do-you-want-to-use-sql-to-query-your-data)
+  - [Do you want better InfluxQL performance?](#do-you-want-better-influxql-performance)
+  - [Do you have requirements for cloud providers and regions?](#do-you-have-requirements-for-cloud-providers-and-regions)
+  - [Are you reliant on Flux queries and Flux tasks?](#are-you-reliant-on-flux-queries-and-flux-tasks)
+- [Data migration guides](#data-migration-guides)
+
+## Should you migrate?
+
+There are important things to consider with migrating to InfluxDB Cloud backed
+by InfluxDB IOx. The following questions will help guide your decision to migrate.
+
+#### Do you not want to worry about series cardinality?
+
+**Yes, you should migrate**. Series cardinality is a major limiting factor with
+the InfluxDB TSM storage engine. The more unique series in your data, the less
+performant your database.
+The IOx storage engine supports near limitless series cardinality and is without
+question, the better solution for high series cardinality workloads.
+
+#### Do you want to use SQL to query your data?
+
+**Yes, you should migrate**. InfluxDB {{< current-version >}} backed by InfluxDB
+IOx lets you query your time series data with SQL. For more information about
+querying your data with SQL, see:
+
+- [Query data with SQL](/influxdb/cloud-iox/...)
+- [InfluxDB SQL reference](/influxdb/cloud-iox/...)
+
+#### Do you want better InfluxQL performance?
+
+**Yes, you should migrate**. One of the primary goals when designing the InfluxDB
+IOx storage engine was to enable performant implementations of both SQL and InfluxQL.
+When compared to querying InfluxDB backed by TSM (InfluxDB OSS 1.x, 2.x, and Enterprise),
+InfluxQL queries are more performant when querying InfluxDB backed by InfluxDB IOx.
+
+#### Do you have requirements for cloud providers and regions?
```

Comment:
this sounds awkward - maybe something like are you dependent on a specific cloud provider or region